File: frontend/test_hooks.js
// Test file for frontend hooks - TESTING SEPARATE HOOKS PER FILE TYPE
function testFunction(name,age) {
  if(age<18){
    return `Hello ${name}, you are ${age} years old and under 18`;
  }else{
    return `Hello ${name}, you are ${age} years old and an adult`;
  }
}

// Intentional formatting issues
const x=1+2+3;
const y=[1,2,3,4,5];
const z={name:"test",value:123};

console.log(testFunction("John",25));

File: backend/test_hooks.py
# Test file for backend hooks - TESTING FIXED HOOKS
def test_function(name: str, age: int) -> str:
    if age < 18:
        return f"Hello {name}, you are {age} years old and under 18"
    else:
        return f"Hello {name}, you are {age} years old and an adult"


# Intentional formatting issues
x = 1 + 2 + 3
y = [1, 2, 3, 4, 5]
z = {"name": "test", "value": 123}

print(test_function("John", 25))
